my_test_hw11.py

```diff
@@ -0,0 +1,185 @@
+from my_func_hw import game_rockscissorspaper
+from my_func_hw import encrypt
+from my_func_hw import Ip
+import unittest
+from unittest.mock import patch
+
+########################################################### unittest - 1
+# class TestHomework1(unittest.TestCase):
```

Comment:
why is this code commented out?

Reply:
there the tests are different (not only the unit test) and so that each one works correctly, I commented out others (it was a bad decision..). You need to uncomment..